core/chains/evm/logpoller/integration_test.go

```diff
@@ -22,6 +26,10 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
+var (
+	EmitterABI, _ = abi.JSON(strings.NewReader(log_emitter.LogEmitterABI))
```

Comment:
Should this error be checked with a Must or something?

Reply:
meh its test code